src/main/java/org/ow2/proactive/catalog/repository/BucketRepositoryImpl.java

```diff
@@ -112,6 +119,14 @@ private CriteriaQuery<Object[]> buildCriteriaQuery(List<String> owners, List<Str
             Predicate ownerPredicate = cb.in(bucketEntityRoot.get("owner")).value(owners);
             allPredicates.add(ownerPredicate);
         }
+
+        if (committedTimeGreater > 0) {
+            allPredicates.add(cb.ge(catalogObjectsJoin.get("lastCommitTime"), committedTimeGreater));
+        }
+        if (committedTimeLessThan > 0) {
+            allPredicates.add(cb.lessThan(catalogObjectsJoin.get("lastCommitTime"), committedTimeLessThan));
+        }
```

Comment:
I believe that you can exclude both filters in case committedTimeLessThan > committedTimeGreater

Reply:
You can't do that using the UI. When you select a start date, the dates before are disabled.
If you force it by manually entering a date in the past, once the form is submitted, it will change to the same date as the start date.
Finally, using the REST api, I personally think that it is confusing to just ignore the filter. Results will be returned and the user can think that these results satisfy the filter. As it is now, you will just get an empty response. 
The other way to handle this in my opinion is to throw an exception